### ISSUE TITLE
White background while loading component

### ISSUE THREAD

**Issue by Thorbenandresen** [wix/react-native-keyboard-input#4]
Hi,

is there a way to supply a color which would be shown instead on the white background while loading a custom keyboard? I sometimes get white flashes when opening/switching custom keyboards, especially if there is a lot going on in the JS thread.

**Comment by artald** [comment]
You can now pass a `backgroundColor` in the `kbInitialProps` of the `KeyboardAccessoryView`.
This color will be used as the background of the native root view.

For example:
`<KeyboardAccessoryView kbInitialProps={backgroundColor: 'black'} />`

Available in version `3.0.1`

**Comment by Thorbenandresen** [comment]
Hey @artald do you have any idea why I get the following error? 

![simulator screen shot apr 21 2017 8 24 09 am](https://cloud.githubusercontent.com/assets/8238976/25279658/2b86fc2c-2675-11e7-892b-025f0637aebe.png)

here is my code

```js

  showKeyboardStickers(component: string, stickers: Model.MessageImages) {


    this.props.updateTracking(component);
    this.setState({
      customKeyboard: {
        component,
        initialProps: {
          stickers,
          backgroundColor: 'black'
        },
      },
    });
  }
```

FYI - this is a problem with my code only. demo app works fine.

**Comment by artald** [comment]
It was a mistake on my part, now fixed in `v3.0.7`. Thanks for reporting about it.
